content/kubermatic/v2.21/architecture/compatibility/supported-versions/_index.en.md

```diff
@@ -31,12 +31,12 @@ these migrations.
 In the following table you can find the supported Kubernetes versions for the
 current KKP version.
 
-| KKP version  | 1.24[^1] | 1.23[^1] | 1.22[^2] | 1.21[^2] | 1.20[^2] | 1.19[^2]   |
-| ------------ | -------- | -------- | -------- | -------- | -------- | ---------- |
-| 2.21.x       | ✓        | ✓        | ✓        | ✓        | -        | -          |
-| 2.20.x       | -        | ✓[^4]    | ✓        | ✓        | ✓        | -          |
-| 2.19.x       | -        | -        | ✓        | ✓        | ✓        | -          |
-| 2.18.x[^3]   | -        | -        | ✓        | ✓        | ✓        | ✓          |
+| KKP version  | 1.26 | 1.25 |1.24[^1] | 1.23[^1] | 1.22[^2] | 1.21[^2] | 1.20[^2] | 1.19[^2] |
+| ------------ | ---- | ---- | ------- | -------- | -------- | -------- | -------- | -------- |
+| 2.21.x       | -    | -    | ✓       | ✓        | ✓        | -        | -        | -        |
```

Comment:
I wanted to keep this up to date to clearly communicate that we're aware of those versions. Personally, I never know if a tool has support for a specific Kubernetes version if it doesn't list it because it might have, but the list wasn't updated for newer releases.

But I can remove that if you want.

Reply:
Okay, that's a good point. It actually gives a clear information about not supporting those versions instead of leaving the readers in confusion.